.github/labeler.yml

```diff
@@ -1,34 +1,42 @@
 "review needed :eyes:":
   - src/**/*
+  - public/*
 
 "documentation :book:":
   - README.md
 
 "tooling :wrench:":
   - .github/**/*
   - src/scripts/*
+  - src/lib/*
+  - src/hooks/*
   - netlify.toml
 
 "dependencies :package:":
   - package.json
   - yarn.lock
 
 "internal :house:":
-  - gatsby-browser.js
-  - gatsby-config.js
-  - gatsby-node.js
-  - gatsby-ssr.js
   - .all-contributorsrc
+  - i18n.config.json
+  - next.config.js
+  - next-i18next.config,js
+  - next-sitemap.config.js
+  - tsconfig.json
+  - .nvmrc
+  - .eslintignore
+  - .eslintrc.json
+  - .prettierignore
+  - .prettierrc
```

Comment:
Removed all Gatsby configs and added all new root level configuration files.

Would consider moving `netlify.toml` config here as well

Reply:
> Would consider moving `netlify.toml` config here as well

I'd say yes.